.dotnet/src/Generated/Models/InternalCreateModerationResponseResultCategoryAppliedInputTypesSelfHarm1.cs

```diff
@@ -0,0 +1,36 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace OpenAI.Moderations
+{
+    internal readonly partial struct InternalCreateModerationResponseResultCategoryAppliedInputTypesSelfHarm1 : IEquatable<InternalCreateModerationResponseResultCategoryAppliedInputTypesSelfHarm1>
```

Comment:
Curious: Do you know why this one is suffixed with a "1" while the others are not?

Reply:
Not entirely sure, but I think it has to do with the array item emplacement making things confused. This was mitigated in this specific instance with the spec update to extract/customize, though.